src/constants/url.ts

```diff
@@ -1,4 +1,6 @@
-export const TINY_API_URL = 'https://staging-tinysite-api.realdevsquad.com/v1';
-export const TINY_API_GOOGLE_LOGIN = `${TINY_API_URL}/auth/google/login`;
-export const TINY_API_LOGOUT = `${TINY_API_URL}/auth/logout`;
-export const BASE_SHORT_URL = 'https://staging-tinysite.realdevsquad.com';
+const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL || 'https://staging-tinysite-api.realdevsquad.com/v1';
+export const BASE_SHORT_URL = process.env.NEXT_PUBLIC_BASE_SHORT_URL || 'https://staging-tinysite.realdevsquad.com';
+
+export const TINY_API_URL = API_BASE_URL;
```

Comment:
what's the need of this extra variable?

Reply:
TINY_API_URL variable is used in multiple places and I want to avoid increasing file changes. I will refactor this in a new full request.

Reply:
You can do this instead
`export const TINY_API_URL =  process.env.NEXT_PUBLIC_API_BASE_URL || 'https://staging-tinysite-api.realdevsquad.com/v1';`

Reply:
It would be better to use API_BASE_URL as the variable name instead of TINY_API_URL.

Reply:
That's why I changed the name to API_BASE_URL.